cmd/cluster/health.go

```diff
@@ -22,6 +23,7 @@ import (
 	cmdutil "k8s.io/kubectl/pkg/cmd/util"
 )
 
+//This command requires the ocm API Token https://cloud.redhat.com/openshift/token be available in the OCM_TOKEN env variable.
 // healthOptions defines the struct for running health command
```

Comment:
this sort-of breaks the godoc cormatting
https://go.dev/blog/godoc
if this can be formatted a bit differently that would be great
```suggestion
// healthOptions defines the struct for running health command
// This command requires the ocm API Token https://cloud.redhat.com/openshift/token be available in the OCM_TOKEN env variable.

```

Reply:
the newline is my bad..
```
go doc -all cmd/cluster
```
won't display the right info with the newline there

you can test this on you machine or I can show you a quick `main.go` of how `go doc` behaves